### PR TITLE
[4.0] com_content author column

### PR DESCRIPTION
**Pull request by brianteeman**
This PR started off just removing the title "author" which is meaningless and not needed.

I then fixed some codestyle issues (spaces to tabs) 

I then simplified the if/else statements

Other than the now missing word Author being displayed on hovering the author name there should be no visible differences

### CODE REVIEW

File: administrator/components/com_content/tmpl/featured/default.php [hunk 1]
```diff
@@ -321,21 +321,17 @@
 									<?php echo $this->escape($item->access_level); ?>
 								</td>
 								<td class="small d-none d-md-table-cell">
-									<?php if ((int) $item->created_by != 0) : ?>
+								<?php if ((int) $item->created_by != 0) : ?>
```
Comment (Quy) [inline administrator/components/com_content/tmpl/featured/default.php]:
```suggestion
									<?php if ((int) $item->created_by != 0) : ?>
```

**Comment by Quy** [comment]
To simplify it further:
```
<?php if ((int) $item->created_by != 0) : ?>
        <a href="<?php echo Route::_('index.php?option=com_users&task=user.edit&id=' . (int) $item->created_by); ?>">
                <?php echo $this->escape($item->author_name); ?>
        </a>
<?php else : ?>
        <?php echo Text::_('JNONE'); ?>
<?php endif; ?>
<?php if ($item->created_by_alias) : ?>
        <div class="smallsub"><?php echo Text::sprintf('JGLOBAL_LIST_ALIAS', $this->escape($item->created_by_alias)); ?></div>
<?php endif; ?>
``

**Comment by brianteeman** [comment]
good idea

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on b95d4f5a324aa452ad72952bdb36671a7a64f502<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25214">issues.joomla.org/tracker/joomla-cms/25214</a>.</sub>

**Comment by richard67** [comment]
@brianteeman The "Author" title indeed was useless. But in the same views, the links to the categories have title "Edit category". So maybe the title of the link to the author should have been changed to "Edit user" instead just removing it? Now it is a bit inconsistent, one kind of links (category) has title, other (author) has not. Do you plan to work on that with a future PR, e.g. remove the other title, too, and maybe addd tool tips to all those links? In this case, if it will be changed anyway, I would give this PR here a good test despite of the new inconsistence. Code analysis looks good, real test shows no changes except of the removed title.

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on b95d4f5a324aa452ad72952bdb36671a7a64f502<br /><br />Despite of the inconsistency that one kind of links (category) has a title and the other one (author) has not after this PR, this PR does what it shall do.
I guess other link titles will be handled with other PRs.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25214">issues.joomla.org/tracker/joomla-cms/25214</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by roland-d** [comment]
I do think that @richard67 has a point here. All the other titles say `Edit ...` , so why just not change the title to `Edit Author`? Unless the idea is to remove all the other labels as well.

**Comment by brianteeman** [comment]
The logic behind my thinking on this is as follows.
This is a list of articles and if you were looking to change the author of an article on an article then you would rightly assume that a link called "edit author" would let you do that. But that's not what the link does. The links lets you see the details of the author and change those details. It does not let you change the author of the article. 
I assume that this logic is why it originally just said "Author"
I removed the title of author as it is "useless" extra information as we know it is an author because it is a column called author

**Comment by richard67** [comment]
Hmm, yes, Brian is right, it would be misleading.

File: administrator/components/com_content/tmpl/featured/default.php [hunk 2]
```diff
@@ -322,21 +322,14 @@
 								</td>
 								<td class="small d-none d-md-table-cell">
 									<?php if ((int) $item->created_by != 0) : ?>
-										<?php if ($item->created_by_alias) : ?>
-											<a class="hasTooltip" href="<?php echo Route::_('index.php?option=com_users&task=user.edit&id=' . (int) $item->created_by); ?>" title="<?php echo Text::_('JAUTHOR'); ?>">
-												<?php echo $this->escape($item->author_name); ?></a>
-											<div class="smallsub"><?php echo Text::sprintf('JGLOBAL_LIST_ALIAS', $this->escape($item->created_by_alias)); ?></div>
```
Comment (wilsonge) [inline administrator/components/com_content/tmpl/featured/default.php]: This smallsub line appears to have been removed too

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**